Dockerfile.tools

```diff
@@ -36,8 +36,12 @@ RUN git clone --branch release-4.6 --single-branch https://github.com/openshift/
 WORKDIR /build/ovn-kubernetes/go-controller/
 RUN make windows
 
-# TODO: Add building the CNI plugins to dockerfile and remove downloading binary
-# https://issues.redhat.com/browse/WINC-490
+# Build CNI plugins
+WORKDIR /build/
+RUN git clone --branch release-4.6 --single-branch https://github.com/openshift/containernetworking-plugins
```

Comment:
I think we should clone WMCO and use the sub-module there. We need to do this for other components too but that can be a separate PR.

@LorbusChris maybe you can tackle this as part of updating sub-modules in WMCO.